R/tm_g_swimlane.R

```diff
@@ -473,8 +473,7 @@ srv_g_swimlane <- function(id,
             }),
             marker_shape_opt = .(if (length(marker_shape_var) == 0) {
               NULL
-            } else if (length(marker_shape_var) > 0 &
-              all(unique(anl[[marker_shape_var]]) %in% names(marker_shape_opt)) == TRUE) { # nolint
+            } else if (length(marker_shape_var) > 0 & all(unique(anl[[marker_shape_var]]) %in% names(marker_shape_opt))) { # nolint: line_length.
```

Comment:
Alternative to consider

```suggestion
            } else if (
              length(marker_shape_var) > 0 &
                all(unique(anl[[marker_shape_var]]) %in% names(marker_shape_opt))
            ) {
```

Reply:
I would rather not grapple with styler changing the indention level.
Btw, this should be `&&`, not `&`.